src/scripts/main.js

```diff
@@ -1,5 +1,26 @@
 'use strict';
 
-document.addEventListener('click', (e) => {
-  // write code here
+document.querySelector('.wall').addEventListener('click', (e) => {
+  const spider = document.querySelector('.spider');
+  const wall = e.currentTarget;
+
+  // Отримуємо координати стіни та розміри павука
+  const wallRect = wall.getBoundingClientRect();
+  const spiderWidth = spider.offsetWidth;
+  const spiderHeight = spider.offsetHeight;
+
+
+  const positionX = e.clientX - wallRect.left + window.pageXOffset;
+  const positionY = e.clientY - wallRect.top + window.pageYOffset;
```

Comment:
The calculation for `positionX` and `positionY` is correct in terms of handling scrolling, but ensure that the spider's center aligns with the click position by adjusting for its width and height. This is done correctly in lines 21 and 22.